xml/gl.xml

```diff
@@ -47740,7 +47740,7 @@ typedef unsigned int GLhandleARB;
                 <enum name="GL_TEXTURE_2D_STACK_BINDING_MESAX"/>
             </require>
         </extension>
-        <extension name="GL_MESA_framebuffer_flip_x" supported="gles2">
+        <extension name="GL_MESA_framebuffer_flip_x" supported="gl|gles2">
```

Comment:
If this extension is also supported in core profile, you have to manually specify it.

```suggestion
        <extension name="GL_MESA_framebuffer_flip_x" supported="gl|glcore|gles2">
```

Reply:
Added.